### PR TITLE
Feature/e UI 8481

### PR DESCRIPTION
**Pull request by MateiHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-8481

### Change description ###
EUI-8481

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/read-query-management-field.component.ts [hunk 1]
```diff
@@ -15,12 +15,14 @@ export class ReadQueryManagementFieldComponent extends AbstractFieldReadComponen
...
   public query: QueryListItem;
   public showQueryList: boolean = true;
+  public caseId: string;
 
   constructor(private readonly route: ActivatedRoute) {
     super();
   }
 
   public ngOnInit(): void {
+    this.caseId = this.route.snapshot.params.cid;
```
Comment (DavidJayakumar) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/read-query-management-field.component.ts:25]: Should we use case notifier service to get the case id? @Daniel-Lam or @ronaldmansveld can you confirm?

**Review by DavidJayakumar — COMMENTED** [review]
@MateiHMCTS Added a comment

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/read-query-management-field.component.ts (hunk 1 above)
Comment (MateiHMCTS) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/read-query-management-field.component.ts:25]: cid is already in the path params; I think getting it from CaseNotifier is an overkill

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/read-query-management-field.component.html [hunk 2]
```diff
@@ -1,9 +1,35 @@
...
+    [query]="query"
+    (backClicked)="showQueryList = true"
+    [caseId]="caseId"
+  ></ccd-query-details>
+
+  <ng-container *ngIf="query?.children?.length > 0; else queryIsInReview">
+    <button id="ask-follow-up-question" class="govuk-button" data-module="govuk-button"
+            [routerLink]="['/query-management', 'query', caseId, '4']"
+    >
+      {{ 'Ask a follow-up question' }}
```
Comment (Daniel-Lam) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/read-query-management-field.component.html:25]: @DavidJayakumar - this needs to be run through the `rpxTranslate` pipe

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@DavidJayakumar - one minor change required

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/read-query-management-field.component.html (hunk 2 above)
Comment (DavidJayakumar) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/read-query-management-field.component.html]: Done

**Review by Daniel-Lam — APPROVED** [review]

**Review by udaydenduluri33 — APPROVED** [review]

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Pull request merged**